nilearn/plotting/displays.py

```diff
@@ -527,10 +527,10 @@ def _map_show(self, img, type='imshow', resampling_interpolation='continuous', *
 
             data_2d_list.append(data_2d)
 
-        if 'vmin' not in kwargs:
+        if kwargs.get('vmin') is None:
```

Comment:
Actually it took me a little while to remember why I did that but this is supposed to catch this two cases:
* vmin has not be passed into the kwargs
* vmin=None has been passed into the kwargs